istio/istio.go

```diff
@@ -370,7 +370,29 @@ func (iClient *IstioClient) executeBookInfoInstall(ctx context.Context, arReq *m
 	return nil
 }
 
-// ApplyRule is a method invoked to apply a particular operation on the mesh in a namespace
+func (iClient *IstioClient) executeBookInfoInstall(ctx context.Context, arReq *meshes.ApplyRuleRequest) error {
+
+	bs, err := ioutil.ReadFile("/home/subham/istio-manifests.yaml")
```

Comment:
you should probably replace this with the contents of the new file `getting_microservices-release.go` (OR) keep that and get rid of this method.